tests/Jellyfin.Naming.Tests/Video/VideoResolverTests.cs

```diff
@@ -1,275 +1,205 @@
+                    {
+                        Path = @"/server/Movies/Brave (2007)/Brave (2006).bluray.disc",
+                        Container = "disc",
+                        Name = "Brave",
+                        Year = 2006,
+                        IsStub = true,
+                        StubType = "bluray",
+                    }
+                };
+                yield return new object[]
+                {
+                    new VideoFileInfo()
+                    {
+                        Path = @"/server/Movies/300 (2007)/300 (2006).bluray.disc",
+                        Container = "disc",
+                        Name = "300",
+                        Year = 2006,
+                        IsStub = true,
+                        StubType = "bluray",
+                    }
+                };
+                yield return new object[]
+                {
+                    new VideoFileInfo()
+                    {
+                        Path = @"/server/Movies/300 (2007)/300 (2006)-trailer.mkv",
+                        Container = "mkv",
+                        Name = "300",
+                        Year = 2006,
+                        ExtraType = ExtraType.Trailer,
+                    }
+                };
+                yield return new object[]
+                {
+                    new VideoFileInfo()
+                    {
+                        Path = @"/server/Movies/Brave (2007)/Brave (2006)-trailer.mkv",
+                        Container = "mkv",
+                        Name = "Brave",
+                        Year = 2006,
+                        ExtraType = ExtraType.Trailer,
+                    }
+                };
+                yield return new object[]
+                {
+                    new VideoFileInfo()
+                    {
+                        Path = @"/server/Movies/300 (2007)/300 (2006).mkv",
+                        Container = "mkv",
+                        Name = "300",
+                        Year = 2006
+                    }
+                };
+                yield return new object[]
+                {
+                    new VideoFileInfo()
+                    {
+                        Path = @"/server/Movies/Bad Boys (1995)/Bad Boys (1995).mkv",
+                        Container = "mkv",
+                        Name = "Bad Boys",
+                        Year = 1995,
+                    }
+                };
+                yield return new object[]
+                {
+                    new VideoFileInfo()
+                    {
+                        Path = @"/server/Movies/Brave (2007)/Brave (2006).mkv",
+                        Container = "mkv",
+                        Name = "Brave",
+                        Year = 2006,
+                    }
+                };
+            }
+
+            IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
+        }
+
+        [Theory]
+        [ClassData(typeof(ResolveFileTestData))]
```

Comment:
`[MemberData]` would probably be more concise in this case. You can use a method returning `IEnumerable<>` instead of defining an entire class